Source/driver/Castro_setup.cpp

```diff
@@ -223,6 +223,19 @@ Castro::variableSetUp ()
 
   init_prob_parameters();
 
+  // check to make sure that we didn't set any parameters that don't
+  // exist in C++ (like because of misspelling).  All of the problem.*
+  // parameters should have been accessed via parmparse at this point.
+
+  if (ParmParse::hasUnusedInputs("problem")) {
```

Comment:
I'm fine to do this; but the code already prints out the full list of unused ParmParse variables at the end of the run. Do you think there's value in adding this additional warning on top of that?

Reply:
yes, because sometimes we never run things to the "end", aka, for big jobs that we just keep restarting.
I'd actually like to make it abort in debug mode -- then we have an easy test